Makefile

```diff
@@ -34,7 +34,7 @@ ifdef TRAVIS_BUILD_NUMBER
 else
 ifndef TIMEOUT
 	# You should be able to specify your own timeout, but by default we'll use the same bound as travis
-	TIMEOUT = 1000s
+	TIMEOUT = 3600s
```

Comment:
Using the old timeout ... the tests don't complete for me locally, so upping to match CI.